sentinel/tests/unit/lib/utils.js

```diff
@@ -85,7 +85,7 @@ describe('utils util', () => {
         .should.equal(66);
     });
     it('throws an exception if the expression errors', () => {
-      should.Throw(() => utils.evalExpression(`doc.foo.bar.smang === 'cats'`, {}));
+      should.Throw(() => utils.evalExpression(`doc.foo.bar.smang === 'cats'`, {})); /* eslint-disable-line new-cap */
```

Comment:
Why does this line require disabling `new-cap`?

Reply:
`should.Throw(()`
ESLint -> A function with a name starting with an uppercase letter should only be used as a constructor. [new-cap]

Reply:
It might work to replace this with...

```js
expect(() => utils.evalExpression(`doc.foo.bar.smang === 'cats'`, {})).to.throw();
```